agent_api_rest/src/holder/mod.rs

```diff
@@ -0,0 +1,27 @@
+// TODO: further refactor the API's folder structure to reflect the API's routes.
```

Comment:
Could there be conflicts or inconsistencies if we follow that strategy (reflecting API routes with folder structure)? It's not uncommon, but does it also apply to this project?

Reply:
We can come up with our own logic and stay consistent to that. I think a nice structure would be to keep all the routes split between domain (issuance, holder etc) and another split between the UniCore API and oid4vc/well-known endpoints. 

Also, how do we structure functions for different methods (GET, POST) to the same endpoints? Do we put them in different files, or do we just name them differently, e.g: `post_credentials`, `get_credentials`.

If there is a conventional way of structuring this I think that can increase consistency. Not sure how there would be conflicts if we do it like this?

Reply:
I opened the following issue for this: https://github.com/impierce/ssi-agent/issues/120